app/facades/users_facade.rb

```diff
@@ -3,4 +3,18 @@ def self.get_user(id)
     user = UsersService.get_user(id)
     User.new(user[:data])
   end
+
+  def self.login_user(omniauth)
+    user_body = { data: {
+      id: omniauth[:id],
```

Comment:
Looking at the example of the authentication hash available in this doc: https://github.com/zquestz/omniauth-google-oauth2 I'm wondering if this needs to be omniauth[:uid] ?

Reply:
So this might have to change once we actually get it implemented with the backend. We adjusted the omniauth mock so that we could actually get an ID to work with in our session.